tomb/src/main.rs

```diff
@@ -52,53 +33,51 @@ async fn main() -> Result<()> {
         // Execute the packing command
         cli::Commands::Pack {
             input_dir,
-            output_dir,
-            chunk_size,
             follow_links,
         } => {
-            match output_dir {
-                Some(output_dir) => {
-                    pack::pipeline(&input_dir, Some(&output_dir), chunk_size, follow_links).await?;
-                },
-                None => {
-                    pack::pipeline(&input_dir, None, chunk_size, follow_links).await?;
-                },
+            if let Some(input_dir) = input_dir {
+                pack::pipeline(&input_dir, follow_links).await?;
+            }
+            else {
```

Comment:
Little nitpick but same line should be preferred here (cargo fmt will fixup all these kinds of things)